packages-exp/auth-exp/src/api/authentication/sign_up.test.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect, use } from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { signUp, SignUpRequest } from './sign_up';
+import { Endpoint } from '..';
+import { ServerError } from '../errors';
+import { FirebaseError } from '@firebase/util';
+import * as mockFetch from '../../../test/mock_fetch';
+import { mockEndpoint, mockAuth } from '../../../test/api/helper';
+
+use(chaiAsPromised);
+
+describe('signUp', () => {
```

Comment:
@samhorlbeck do you think this is sufficient coverage per endpoint? or should we test more? most of the endpoints are parameterized so there isn't much going on

Reply:
Yeah I think it's fine. There's almost no actual logic in these, it's mostly type scaffolding